plugins/GSdx/GSState.cpp

```diff
@@ -1265,11 +1265,6 @@ void GSState::GIFRegHandlerSCISSOR(const GIFReg* RESTRICT r)
 template <int i>
 void GSState::GIFRegHandlerALPHA(const GIFReg* RESTRICT r)
 {
-	ASSERT(r->ALPHA.A != 3);
```

Comment:
Maybe replace them with a log instead?

Reply:
I don't really think the info is all that useful but I can stick a GL_REG in there if that makes you feel better.

Reply:
Yeah it would be nice to have it.

Reply:
done.